Apsitvarkom.Models/DTO/LocationDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Apsitvarkom.Models.DTO;
+
+/// <summary>
+/// Data Transfer Object for <see cref="Location" />
+/// </summary>
+public class LocationDTO
```

Comment:
Consider making this a struct? I think one of the requirements for lab 1 is that we know how to create structs 😄

Reply:
It would make sense here, since we only have 2 primitive fields